src/patterns/molecules/forms/container.hbs

```diff
@@ -0,0 +1,70 @@
+---
+name: Button Container
+description: |
+  The `.mzp-c-button-container` class can be used around a group of buttons. It applies a flexbox context and the default   form spacing. Modifier classes can then be added to change the layout.
+order: 1
+---
+<div class="mzp-c-form">
+  <h2 class="mzp-u-title-xs">Default</h2>
+  <div class="mzp-c-button-container">
+    {{#embed "patterns.atoms.buttons.primary" }}{{#content "content"}}Primary{{/content}}{{/embed}}
+    {{#embed "patterns.atoms.buttons.secondary" }}{{#content "content"}}Secondary{{/content}}{{/embed}}
+    <p class="mzp-c-button-info">More information.</p>
+  </div>
+
+  <h2 class="mzp-u-title-xs"><code>mzp-l-h-end</code> Align buttons to the right</h2>
+  <div class="mzp-c-button-container mzp-l-h-end">
+    {{#embed "patterns.atoms.buttons.primary" }}{{#content "content"}}Primary{{/content}}{{/embed}}
+    {{#embed "patterns.atoms.buttons.secondary" }}{{#content "content"}}Secondary{{/content}}{{/embed}}
+    <p class="mzp-c-button-info">More information.</p>
+  </div>
+
+  <h2 class="mzp-u-title-xs"><code>mzp-l-h-center</code> Align buttons in the center</h2>
+  <div class="mzp-c-button-container mzp-l-h-center">
+    {{#embed "patterns.atoms.buttons.primary" }}{{#content "content"}}Primary{{/content}}{{/embed}}
+    {{#embed "patterns.atoms.buttons.secondary" }}{{#content "content"}}Secondary{{/content}}{{/embed}}
+    <p class="mzp-c-button-info">More information.</p>
+  </div>
+
+  <h2 class="mzp-u-title-xs"><code>mzp-l-h-between</code> Space buttons appart evenly, starting from the edges of the container</h2>
+  <div class="mzp-c-button-container mzp-l-h-between">
+    {{#embed "patterns.atoms.buttons.primary" }}{{#content "content"}}Primary{{/content}}{{/embed}}
+    {{#embed "patterns.atoms.buttons.secondary" }}{{#content "content"}}Secondary{{/content}}{{/embed}}
+    <p class="mzp-c-button-info">More information.</p>
+  </div>
+
+  <h2 class="mzp-u-title-xs"><code>mzp-l-column</code> Stack the buttons in one column</h2>
+  <div class="mzp-c-button-container mzp-l-column">
+    {{#embed "patterns.atoms.buttons.primary" }}{{#content "content"}}Primary{{/content}}{{/embed}}
+    {{#embed "patterns.atoms.buttons.secondary" }}{{#content "content"}}Secondary{{/content}}{{/embed}}
+    <p class="mzp-c-button-info">More information.</p>
+  </div>
+
+  <h2 class="mzp-u-title-xs"><code>mzp-l-column-xs-sm</code> Stack the buttons in one column on xs and sm screens</h2>
+  <div class="mzp-c-button-container mzp-l-column-xs-sm">
```

Comment:
This should be `mzp-l-column-on-xs-sm` with the "on" (assuming we go with that convention... or whatever convention we go with, but the HTML should match the CSS 😉)